codenav/src/codenavigation.h

```diff
@@ -42,7 +42,6 @@
 
 /* Last */
 #include "plugindata.h"		/* this defines the plugin API */
-#include "geanyfunctions.h"	/* this wraps geany_functions function pointers */
```

Comment:
yeah that probably would be better, esp. as it's misused :)

We may want to deprecated `utils_str_equal()` in favor to this at the same time.